README.md

```diff
@@ -1,9 +1,20 @@
-[![Build Status](https://travis-ci.org/oasis-roles/wipefs.svg?branch=master)](https://travis-ci.org/oasis-roles/wipefs)
+[![Build Status](https://travis-ci.org/oasis-roles/disk_wipe.svg?branch=master)](https://travis-ci.org/oasis-roles/disk_wipe)
 
-wipefs
+disk_wipe
 ===========
 
-Basic description for wipefs
+This role is not idempotent and is intended to be used when you are preparing nodes
+for deployment to ensure there is no stale data from previous deployments.
```

Comment:
Stale data from previously deployments is only one possibility.  It's possible to buy a brand new disk and have it contain partitions directly from the manufacturer.

Reply:
updated